src/library/basesqltablemodel.cpp

```diff
@@ -38,18 +38,12 @@ const QString kEmptyString = QStringLiteral("");
 
 } // anonymous namespace
 
-BaseSqlTableModel::BaseSqlTableModel(
-        QObject* parent,
-        TrackCollectionManager* pTrackCollectionManager,
-        const char* settingsNamespace)
-        : BaseTrackTableModel(
-                  settingsNamespace,
-                  pTrackCollectionManager,
-                  parent),
-          m_pTrackCollectionManager(pTrackCollectionManager),
-          m_database(pTrackCollectionManager->internalCollection()->database()),
-          m_bInitialized(false),
-          m_currentSearch(kEmptyString) {
+BaseSqlTableModel::BaseSqlTableModel(QObject* parent, TrackCollectionManager* pTrackCollectionManager, const char* settingsNamespace)
```

Comment:
This line of code exceeds the GitHub limit. Please break.
Did clang-format unbreak it?  It can be tricked by manually breaking after (. 
I think the recent pre-commit hook can do it as well. 
Please verify the whole PR.

Reply:
I tried to get the code somewhat consistent and readable, but yeah I also wasn't quite sure here. I'll adjust it.